src/main/java/frc/robot/RobotContainer.java

```diff
@@ -41,6 +42,7 @@ public class RobotContainer {
 
   // The robot's subsystems and commands are defined here...
   private final IntakeSubsystem intakeSubsystem = new IntakeSubsystem();
+  private final ShooterTriggerSubsystem shooterTriggerSubsystem = new ShooterTriggerSubsystem();
```

Comment:
Honest question: why instantiate this here, as opposed to in the constructor?

Reply:
Mistype copied from intake